### PR TITLE
Fix for: #34 (README.md always references Apache License ignoring freshLicence) 

### PR DESCRIPTION
**Pull request by strobe**
just a quick fix of #34, may be it required additional effort to be polished

### CODE REVIEW

**Comment by behaghel** [comment]
That was quick! 👏

File: src/main/scala/de/heikoseeberger/sbtfresh/Template.scala [hunk 1]
```diff
@@ -252,7 +260,10 @@ private object Template {
        |addSbtPlugin("de.heikoseeberger" % "sbt-header"   % "1.6.0")
        |""".stripMargin
 
-  def readme(name: String): String =
+  def readme(name: String, license: String): String = {
+    val licenseKind: License.EnumVal = getLicenseKind(license)
```
Comment (hseeberger) [inline src/main/scala/de/heikoseeberger/sbtfresh/Template.scala]: Do we need this assignment? Maybe just call `getLicensKind(license)` inline in the next line.

File: src/main/scala/de/heikoseeberger/sbtfresh/Template.scala [hunk 2]
```diff
@@ -252,7 +260,10 @@ private object Template {
...
+    val licenseMetaData: String = readmeLicenseMetaData.get(licenseKind).getOrElse("[Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html)")
```
Comment (hseeberger) [inline src/main/scala/de/heikoseeberger/sbtfresh/Template.scala]: No type annotations for local variables.

Comment (hseeberger) [inline src/main/scala/de/heikoseeberger/sbtfresh/Template.scala]: Also, use `License.ApacheV2` instead of the string.

**Pull request merged**